packages/palette/src/elements/Input/Input.tsx

```diff
@@ -20,6 +23,9 @@ export interface InputProps
   hover?: boolean
   required?: boolean
   title?: string
+  prefixOffset?: number
+  suffixOffset?: number
```

Comment:
I feel strongly that we need to get rid of these props. We extend BoxProps so just use them — I'm confused by this

Reply:
These were added to support `LabeledInput` since we used to have a default offset the label that was only added to the right of the element


In the new designs we have two options for labels, either left or right, and since we use our `Input` for it, we need to add an offset for the input and for the label positioning if the label is on the left

This is where it is used: [code](https://github.com/artsy/palette/blob/mrsltun/CX-3330/new-input-component/packages/palette/src/elements/LabeledInput/LabeledInput.tsx#L36)

Reply:
I'd probably just split off `pl` and `pr` though? We don't want these exposed publicly. We could also dunder name them to discourage usage.

Reply:
Why accept arbitrary values if we know the exact contexts in which they are used?

Reply:
Cool! Does the `labelOffset` actually change? Or can it just be a boolean?

Reply:
It's a number that adds a padding-left to the label, and that number depends on the width of the text in the `LabeledInput`

Reply:
Ah OK — got it thanks